src/styles/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
```

Comment:
Not resolved. Just try to center the calendar vertically so that absolutely nothing goes beyond the screen, If you have any questions - feel free to ask them in `fe_chat`.